src/store/callViewStore.js

```diff
@@ -86,6 +90,16 @@ const mutations = {
 	clearParticipantHandRaised(state) {
 		state.participantRaisedHands = {}
 	},
+	initializeBlurredBackgroundImageCache(state, videoBackgroundId) {
+		if (state.blurredBackgroundImageCache[videoBackgroundId]) {
+			return
+		}
+
+		Vue.set(state.blurredBackgroundImageCache, videoBackgroundId, new LRU({ max: 10 }))
```

Comment:
so what happens in a call with 12 users and they speak all one sentence after another, it will have to calculate again and again?

Reply:
No, each participant has (or should :-P ) its own cache; 10 is the number of backgrounds for each participant (as the blurred backgrounds are different depending on the size of the video background).